default/stringtables/en.txt

```diff
@@ -1248,6 +1248,18 @@ Toggles whether to draw circles around systems.
 OPTIONS_DB_UI_SYSTEM_CIRCLE_SIZE
 Sets size of circles around systems on map, relative to system icon size.
 
+OPTIONS_DB_UI_SYSTEM_INNER_CIRCLE_WIDTH
+Sets the line width of the inner system circle when the map is zoomed in.
+
+OPTIONS_DB_UI_SYSTEM_OUTER_CIRCLE_WIDTH
+Sets the line width of the outer system circle.
+
+OPTIONS_DB_UI_SYSTEM_INNER_CIRCLE_MAX_WIDTH
```

Comment:
This doesn't have an effect when the map is zoomed in. But when you zoom out, systems with colonies can be identified by their thicker circle.